docs/data/destinations/appsflyer.md

```diff
@@ -33,12 +31,69 @@ To complete setup, you need the following information from AppsFlyer:
 - REST API Key: Find this in **App Settings > Dev Key**.
 - AppsFlyer App ID: The app identifier used in your AppsFlyer dashboard. It also appears in the URL when you are on the dashboard.
 
-There are no other setup steps in AppsFlyer. 
+### Amplitude setup
+
+1. In Amplitude, navigate to **Data**.
+2. On the sidebar, under **Connections**, select **Catalog**.
+3. Select the **Destinations** tab.
+4. Select **Event Streaming**.
+5. Click **Appsflyer - Event Streaming**.
+6. Enter a sync name, then click **Create Sync**.
+
+After you create the destination, you must configure the settings.
+
+#### Configure settings
+
+1. On the **Settings** tab, click **Edit**.
+2. Enter your **Appsflyer App ID**.
+3. Enter your **REST API Key**.
+4. **Send Events** sends events ingested by Amplitude to Appsflyer.
+      1. To send an event, toggle **Send Events** to **Enabled**.
+      2. Expand the **Select and filter events** panel, and select which events to send.
+      3. To select event properties to send, expand the **Specify event properties to send** panel, and select properties you want to include. If you don't select any properties here, Amplitude doesn't send any.
+5. Save when finished.
+
+After you configure your settings, configure your mappings.
+
+#### Configure mappings (recommended)
+
+For newer versions of the Appsflyer destination, you can map Amplitude properties to fields in Appsflyer. If you don't configure mappings, or are using an older version that doesn't support mapping, the default values described below are used instead.
+
+1. Click the **Mappings** tab, then click **Edit**.
+2. Select a **Appsflyer ID**. This is a unique identifier for users in Appsflyer. It defaults to Amplitude **User ID** (`user_id`).
+3. Select a **Customer User ID**. This is a different unique identifier for users in Appsflyer that you can configure. It also defaults to Amplitude **User ID** (`user_id`) in older syncs.
```

Comment:
NIT: How much do we care about minor consistency in wording across similar docs @caseyamp? In these docs we say `It defaults to...` while GA4 docs just say `Defaults to...`.

Reply:
I'd prefer to use "It defaults to.." when it's in a sentence, and "Defaults to" when it's in a parameter table or similar. 
I think we're fine keeping "it" here